user/troubleshooting/resume-suspend-troubleshooting.md

```diff
@@ -132,3 +134,8 @@ This can be achieved under a Windows HVM by opening the Device Manager, selectin
 This is illustrated on the screenshot below:
 
 ![r2b1-win7-usb-disable.png](/attachment/wiki/HvmCreate/r2b1-win7-usb-disable.png)
+
+## Suspend turns off the screen and gets stuck
+
+On some devices suspend-to-RAM does not work and a hard power-off is needed to recover, because the system does not go into deep sleep. To get suspend to work, you need to add `mem_sleep_default=deep` to the every `kernel=` line in `/boot/efi/EFI/qubes/xen.cfg` and reboot.
```

Comment:
The `/boot/efi/EFI/qubes/xen.cfg` file applies for UEFI boot on R4.0 only. For legacy boot, or UEFI/legacy on R4.1+, the correct place is `/etc/default/grub` (`GRUB_CMDLINE_LINUX` setting).

Reply:
You are right. I updated the PR accordingly.